rotkehlchen/chain/aggregator.py

```diff
@@ -1168,36 +1170,41 @@ def add_accounts_to_all_evm(
         failed_accounts: list[tuple[SUPPORTED_EVM_EVMLIKE_CHAINS_TYPE, ChecksumEvmAddress]] = []
         existed_accounts: list[tuple[SUPPORTED_EVM_EVMLIKE_CHAINS_TYPE, ChecksumEvmAddress]] = []
         no_activity_accounts: list[tuple[SUPPORTED_EVM_EVMLIKE_CHAINS_TYPE, ChecksumEvmAddress]] = []  # noqa: E501
-        evm_contract_addresses: list[tuple[SUPPORTED_EVM_EVMLIKE_CHAINS_TYPE, ChecksumEvmAddress]] = []  # noqa: E501
+        contract_addresses: list[ChecksumEvmAddress] = []
 
         for account in accounts:
             existed_accounts += [(chain, account) for chain in SUPPORTED_EVM_EVMLIKE_CHAINS if account in self.accounts.get(chain)]  # noqa: E501
-            # Distinguish between contracts and EOAs
             chains_to_check = [x for x in SUPPORTED_EVM_EVMLIKE_CHAINS if account not in self.accounts.get(x)]  # noqa: E501
-            chains_with_valid_addresses = []
+
+            # Check if this address is a contract on any chain (for tagging purposes)
```

Comment:
a question. is the tagging necessary? i'm asking because what happens to previously added safe accounts.